internal/usage/usage.go

```diff
@@ -71,6 +71,8 @@ const (
 	NotificationToken               = "Slack API token or Bot token or Flowdock personal API token." //nolint:gosec // This is just a message not a password
 	NotificationsChannelName        = "Slack channel name. Required for the SLACK notifications type."
 	APIKey                          = "Datadog API Key, Opsgenie API Key, VictorOps API key."
+	APIKeyDescription               = "Description of the API key."
```

Comment:
I think that for description you can reuse `usage.Comment`

Reply:
I'll leave this one based on the above conversation -- `description` seems to actually be mandatory